.tekton/release-pipeline.yaml

```diff
@@ -72,25 +72,26 @@ spec:
                     exit
                 }
                 stable_tag=${version%.*}.x
+                allversions=$(echo -n "nightly,stable,";for i in $(git tag -l|grep '^v'|sort -rn);do echo -n "$i,";done|sed 's/,$//')
+                git config --global user.email "pac-dev@redhat.com"
+                git config --global user.name "Pipelines as Code CI Robot"
+
                 for target in release-${version} release-${stable_tag} stable;do
-                  export PAC_VERSION=${version} TARGET_BRANCH=${target//release-}
+                  export PAC_VERSION=${version}
+                  export TARGET_BRANCH=${target//release-}
                   ./hack/generate-releaseyaml.sh > /tmp/release.k8s.yaml
                   env TARGET_OPENSHIFT=true bash hack/generate-releaseyaml.sh > /tmp/release.yaml
                   msg="pac release ${version} on branch ${target}"
-                  # fetch all versions and generate a file for the dropdown menu to select the version documentation
-                  allversions=$(echo -n "nightly,stable,";for i in $(git tag -l|grep '^v'|sort -rn);do echo -n "$i,";done|sed 's/,$//')
-                  set +x
-                  echo python hack/upload-file-to-github.py \
```

Comment:
After using git commands i think we no more equire `upload-file-to-github.py` file 
if thats the case can we remove that file as well ?

Reply:
sure :+1: